ext/puma_http11/extconf.rb

```diff
@@ -12,4 +12,10 @@
   end
 end
 
+BROKEN_VERSIONS = %w(2.2.7 2.3.4 2.4.1)
```

Comment:
Shouldn't we be fixing this for 2.2.0 through 2.2.6, 2.3.0 through 2.3.3 and 2.4.0 as well?